production/kubernetes/agent-traces.yaml

```diff
@@ -72,10 +72,14 @@ spec:
     port: 9411
     protocol: TCP
     targetPort: 9411
-  - name: grafana-agent-traces-otlp
+  - name: grafana-agent-traces-otlp-grpc
```

Comment:
This file is generated from [Jsonnet](https://github.com/grafana/agent/blob/a16355366a39a66d5b33a626823a000d84e167b0/production/kubernetes/build/templates/traces/main.jsonnet#L18-L31). Could you update the Jsonnet instead and then run this command from the root of the repository to generate the file?

```
USE_CONTAINER=1 make generate-manifests 
```

This assumes you have Go and Docker installed. If you don't have either of those installed (or you're having trouble running the command), I can run it for you after the Jsonnet is updated and push a commit.

Reply:
Okay, I will do this.

Reply:
Thank you for the suggestion. I update Jsonnet and run generate-manifests and the result is the same as agent-traces.yaml, I've done it before.